backend/ventserver/simulation/alarm_mute.py

```diff
@@ -10,20 +11,18 @@
 from ventserver.protocols.backend import states
 from ventserver.protocols.protobuf import mcu_pb
 
-def transform(
-    request: mcu_pb.AlarmMuteRequest,
-    response: mcu_pb.AlarmMute
-) -> None:
-    """Mute the current active alarm"""
-
-    response.active = request.active
-
 @attr.s
 class AlarmMuteService:
     """Implement Alarm Mute Service"""
 
-    def transform(# pylint: disable=no-self-use
-            self, store: Mapping[
+    mute_duration: float = attr.ib(default=0)  # ms after mute_start_time
+    mute_start_time: float = \
+        attr.ib(default=time.time() * 1000)  # ms, Unix time
+    mute_max_duration: int = 120000 # ms
```

Comment:
The convention in the firmware is that all timestamps should be `uint32_t`s in ms, while in the backend all timestamps should be `float`s in s; this is because of the types and units for the firmware's `HAL::Time.millis()`-based clock and the backend's `time.time()`-based clock. It looks like maybe the backend's breathing circuit simulator doesn't follow this convention yet; but if you can follow this convention for `AlarmMuteService`, that'd be great.